views/partials/scan_form.hbs

```diff
@@ -0,0 +1,13 @@
+  <form action="/scan" class="email-scan" method="post">
+      <div class="input-group">
+          <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
+          <input type="hidden" name="emailHash">
+          <div class="input-group-button">
+              <img src="img/Search.svg" alt="search icon" />
+              <img class="loader" src="img/loader.gif" alt="loading data" />
+              <input id="submit-email" type="submit" class="button" value="Search">
+          </div>
+      </div>
+      <span>Your email will not be stored. Find out more in our <a href="
+      https://www.mozilla.org/privacy/firefox/" alt="Mozilla Firefox privacy policy">Privacy Policy.</a></span>
```

Comment:
nit: remove `alt="..."` or change it to some `aria-label="..."` or something if we're doing this fo a11y reasons.